### DISCUSSION TITLE
Trying to use LiveCharts, but I get "Cannot build with both Uno.WinUI and Uno.UI nuget packages referenced"

### DISCUSSION THREAD

**Discussion by n2-holden** [unoplatform/uno#11002] [Q&A]
I'm targeting both Wasm and Windows.  Both project types give the same error when I add LiveChartsCore.SkiaSharpView.Uno

**Comment by jeromelaban** [comment]
Thanks for the report! We'll update our docs for the message to be clearer, but you need to be using WinUI packages. In your case, you'll need to add `LiveChartsCore.SkiaSharpView.Uno.WinUI` instead of `LiveChartsCore.SkiaSharpView.Uno`.
